compiler/noirc_evaluator/src/ssa/opt/unrolling.rs

```diff
@@ -330,6 +748,9 @@ struct LoopIteration<'f> {
     /// This is None until we visit the block which jumps back to the start of the
     /// loop, at which point we record its value and the block it was found in.
     induction_value: Option<(BasicBlockId, ValueId)>,
+
+    /// Whether to discard reference count instructions, which ACIR doesn't need but Brillig does.
+    skip_ref_counts: bool,
```

Comment:
```suggestion
```
We should not need this field with the `inserter`. If we want to reduce the need to call `self.inserter.function.runtime().is_acir()` we can make a method inside of the `LoopIteration` context.

Reply:
> with the `inserter` ... we can make a method inside of the `LoopIteration` context.

To be clear, this is a field added to the `LoopIteration`, as you say so that we don't have to call `.runtime().is_acir()`, but if calling that is not a big overhead I'm happy to remove this 👍